<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>13</horstretch>
     <verstretch>4</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Devices</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableWidget" name="devices">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <column>
           <property name="text">
            <string>Packages</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Manufacturer</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Datasheet</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Packages</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTableWidget" name="packages">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <column>
           <property name="text">
            <string>Module</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Manufacturer</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Datasheet</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Symbols</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="editWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="title">
            <string>Manufacturing</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox_2">
              <item>
               <property name="text">
                <string>A</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>B</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>C</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Standard</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>IPC7351</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Density</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
           <zorder>label</zorder>
           <zorder>comboBox</zorder>
           <zorder>label_2</zorder>
           <zorder>comboBox_2</zorder>
           <zorder>horizontalSpacer</zorder>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Part</string>
            </attribute>
            <layout class="QVBoxLayout" name="vert34">
             <item>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit"/>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="lineEdit_3"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Pins</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>D</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QLineEdit" name="lineEdit_4"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>E1</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>E</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>b</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="4">
                <widget class="QLineEdit" name="lineEdit_6"/>
               </item>
               <item row="3" column="4">
                <widget class="QLineEdit" name="lineEdit_8"/>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_2"/>
               </item>
               <item row="0" column="2" colspan="2">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>S</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>L</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="lineEdit_7"/>
               </item>
               <item row="3" column="2">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>e</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="4">
                <widget class="QLineEdit" name="lineEdit_5"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGraphicsView" name="graphicsView_2"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_4">
            <attribute name="title">
             <string>Silk</string>
            </attribute>
            <widget class="QComboBox" name="comboBox_3">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>10</y>
               <width>82</width>
               <height>33</height>
              </rect>
             </property>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Diode</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dot</string>
              </property>
             </item>
            </widget>
            <widget class="QLabel" name="label_11">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>20</y>
               <width>71</width>
               <height>17</height>
              </rect>
             </property>
             <property name="text">
              <string>Pin marking</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>editWidget</zorder>
   <zorder>tabWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTesting_something">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_DB"/>
    <addaction name="actionSave_DB"/>
    <addaction name="separator"/>
    <addaction name="actionExport_library"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuTesting_something"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_DB">
   <property name="text">
    <string>Open DB</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_DB">
   <property name="text">
    <string>Save DB</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport_library">
   <property name="text">
    <string>Export library</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
